instadam/models/image.py

```diff
@@ -0,0 +1,25 @@
+import datetime as dt
+from ..app import db
+
+
+class Image(db.Model):
+    """Class User is a database model to represent a user
+
+    Specifies the full database schema of the table 'user'
+
+    Attributes:
+        id: unique integer id given to a user (primary key)
+        project_id: integer to represent id of project that this image belongs to
+        image_name: string to represent name of image
+        added_at: datetime that image was added to the project
+    """
+
+    __tablename__ = 'images'
+    id = db.Column(db.Integer, primary_key=True)
+    project_id = db.Column(db.Integer, nullable=False)
```

Comment:
Shouldn't this be a foreign key? https://docs.sqlalchemy.org/en/latest/core/constraints.html